config/opal_case_sensitive_fs_setup.m4

```diff
@@ -78,6 +80,12 @@ else
 fi
 
 AM_CONDITIONAL(CASE_SENSITIVE_FS, test "$OPAL_WANT_CS_FS" = "1")
+# There is a case in the ompi/tools/wrappers/Makefile.am where we need
```

Comment:
I thought Automake let you have if blocks inside if blocks, which seems much nicer.

Reply:
It does, but then I have to repeat the "no" block twice.

E.g.:

```
if A
if B
...YES stuff...
else # B
...NO stuff...
endif # B
else # A
...NO stuff...
endif # A